Cargo.toml

```diff
@@ -27,7 +27,7 @@ ic-stable-structures = { workspace = true }
 ic-certified-map = { workspace = true }
 ic-cdk = { workspace = true }
 ic-cdk-macros = { workspace = true }
-cketh-common = { git = "https://github.com/dfinity/ic", rev = "13af7fa681e3055adae4e83d172204bd3c8188b6", package = "ic-cketh-minter" }
+cketh-common = { git = "https://github.com/rvanasa/ic", rev = "4cf8effe8c2f60f889955302a79a800f898dfb29", package = "ic-cketh-minter" }
```

Comment:
Changing back to the personal branch for now to speed up development (since modifying the `dfinity/ic` branch requires an MR).